src/azure-cli/azure/cli/command_modules/resource/_params.py

```diff
@@ -11,7 +11,8 @@ def load_arguments(self, _):
 
     from azure.mgmt.resource.locks.models import LockLevel
     from azure.mgmt.resource.managedapplications.models import ApplicationLockLevel
-    from azure.mgmt.resource.policy.models import (ExemptionCategory, EnforcementMode)
```

Comment:
Removed the ExemptionCategory import so the CI Builds could pass

Reply:
Context about `ExemptionCategory` and I think you can fix it by the following code instead of delete it:
```python
from azure.mgmt.resource.policy.v2020_07_01_preview.models import ExemptionCategory
from azure.mgmt.resource.policy.models import EnforcementMode
```

Reply:
I added back the `from azure.mgmt.resource.policy.v2022_07_01_preview.models import ExemptionCategory` fix and it seems like the tests are failing again. 
`